joomla/joomla-cms
# Registration model doesn't create correct messages on failed registration email #8900

Pull Request for Issue #8900 .

Correction of message to admins on user registration email send failure.

Testing Instructions:
1. Set dummy SMTP settings in Global Configuration.
2. Register a user, see "Warning
3. An error was encountered while sending the registration email. A message has been sent to the administrator of this site." message.
4. There is no message in /administrator/index.php?option=com_messages
5. Apply patch.
6. Register user, see message in /administrator/index.php?option=com_messages
